util/cron/nightly

```diff
@@ -878,9 +878,9 @@ if ($runtests == 0) {
     }
 
 # FIXME: Pass correct args here!
-     # `$chplhomedir/util/cron/nightly_email.pl $status "$rawsummary" "$sortedsummary" "$prevsummary" "$mailer" "$nochangerecipient" "$recipient" "$subjectid" "$config_name" "$revision" "$rawlog" "$starttime" "$endtime" "$crontab" "$testdirs" $debug`;
+   `$chplhomedir/util/cron/nightly_email.pl $status "$rawsummary" "$sortedsummary" "$prevsummary" "$mailer" "$nochangerecipient" "$recipient" "$subjectid" "$config_name" "$revision" "$rawlog" "$starttime" "$endtime" "$crontab" "$testdirs" $debug`;
```

Comment:
The backtick '\`' character in Perl causes it to run the external command and return the output. In this case, we aren't doing anything with the output, so it just disappears. If you change this to ``print `$chplhome... ``then any output from `nightly_email.pl` will be printed by the `nightly` script, which will help us debug it.